### PR TITLE
IAP-3508 Enhance unattended mode installer/uninstaller to use a config file for parameters

### PR DESCRIPTION
**Pull request by kvedenskii-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Add optionfile video

## Deploy preview link

https://deploy-preview-1903--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/install_unattended/#install-and-uninstall-api-portal-using-a-configuration-file

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by andreamussap** [comment]
@dginevaAxway please share with @kvedenskii-axway and Tesla team the confluence page "Contribute locally directly without a fork", specially section "Update your pull request" so he can create a proper title for the PR, etc. 

Also, @kvedenskii-axway have a look at the list of commits on your PR!

**Comment by kvedenskii-axway** [comment]
> Also, @kvedenskii-axway have a look at the list of commits on your PR!

Sorry about that, made PR to the wrong branch

**Review by dginevaAxway — APPROVED** [review]

File: content/en/docs/apim_installation/apiportal_install/install_unattended.md [hunk 1]
```diff
@@ -202,6 +202,8 @@ This can be useful to avoid adding sensitive data to the configuration file, or
 
 {{< alert title="Note" color="primary" >}}Do not add sensitive information to the configuration file.{{< /alert >}}
 
+{{< youtube HqQ77Cj2s5E >}}
+
 ## Encrypt the Public API user password in unattended mode
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_unattended.md:207]: @kvedenskii-axway what's the introductory text for this video? "Watch this video to learn how to install and uninstall API Portal using a configuration file"?

File: content/en/docs/apim_installation/apiportal_install/install_unattended.md [hunk 2]
```diff
@@ -202,6 +202,8 @@ This can be useful to avoid adding sensitive data to the configuration file, or
 
 {{< alert title="Note" color="primary" >}}Do not add sensitive information to the configuration file.{{< /alert >}}
 
+{{< youtube HqQ77Cj2s5E >}}
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apiportal_install/install_unattended.md:205]: @kvedenskii-axway what's the introductory text for this video? "Watch this video to learn how to install and uninstall API Portal using a configuration file"?

Comment (varlogerr) [reply content/en/docs/apim_installation/apiportal_install/install_unattended.md:207]: Yes, it would be the correct intro text. Updated the PR

**Pull request merged**